app/gui/view/debug_scene/component-group/src/lib.rs

```diff
@@ -12,24 +12,33 @@
 #![warn(unused_import_braces)]
 #![warn(unused_qualifications)]
 
+use ensogl_core::display::shape::*;
 use ensogl_core::prelude::*;
 use wasm_bindgen::prelude::*;
 
 use ensogl_core::application::Application;
 use ensogl_core::data::color;
 use ensogl_core::display::object::ObjectOps;
 use ensogl_core::frp;
-use ensogl_core::Animation;
 use ensogl_hardcoded_theme as theme;
 use ensogl_list_view as list_view;
-use ensogl_selector as selector;
-use ensogl_selector::Bounds;
+use ensogl_scroll_area::ScrollArea;
 use ensogl_text_msdf_sys::run_once_initialized;
 use ide_view_component_group as component_group;
 use list_view::entry::AnyModelProvider;
 
 
 
+mod transparent_circle {
+    use super::*;
+    ensogl_core::define_shape_system! {
+        (style:Style) {
+            Circle(5.px()).fill(color::Rgba::transparent()).into()
+        }
+    }
+}
+
```

Comment:
1. Eveything hould be in section
2. There is no explanation of what is this thing
3. There are hardcoded literals inside without explanation of what they do.

Reply:
1. Moved
2. Added, with a link to the pivotal bug
3. It turned out even 0 as radius works! Added a comment.